lib/jekyll-import/importers/blogger.rb

```diff
@@ -1,3 +1,6 @@
+require 'open-uri'
+require 'reverse_markdown'
```

Comment:
Both of these need to be in `self.require_deps` function inside the `Blogger` class.